_layouts/code_comp_page.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+---
+<head>
+        <title>MD2</title>
```

Comment:
Markdown, second try!
It's invisible and doesn't break anything, but I'll remove it anyway.